common.lic

```diff
@@ -192,8 +192,14 @@ module DRC
 
   def stow_hands
     pause 1
-    bput('stow right', 'You put') if checkright
-    bput('stow left', 'You put') if checkleft
+    stow_hand('right') if checkright
```

Comment:
I wonder if a lot of this logic could be simplified if you called `stow_hand(checkright)` (and checkleft). Then add a nil check to return early to `stow_hand`. That way you pass in the noun and don't need to try to parse it.